docs/finality-provider-operation.md

```diff
@@ -928,8 +929,16 @@ This command should ask to
 `confirm transaction before signing and broadcasting [y/N]:` and output the
 transaction hash.
 
-Once the address is set, rewards can be withdrawn by running the
-following command:
+#### 6.7.3 Withdraw Rewards
+
+Once the address is set, be sure to set the operational key name in the
+keyring home directory in the `[babylon]` config in `fpd.conf`.
+
+> ⚠️ **Important**: The `fpd` must be stopped before performing this function.
+
+Once the `fpd` is stopped, run the following to withdraw rewards.
```

Comment:
The reason this could be encountered is due to using the registered key as the operational key. If the user follows 6.8, they don't need to stop `fpd`. We can add this note with reference here